docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -115,59 +115,69 @@ If you see a version number, Homebrew has been successfully installed.
 
     b. Copy and paste the following command, then press `Enter`:
 
-    ```
+    ```bash
     brew install kind
     ```
 
     c. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
 
 This is the end of the one-time installations and the following steps will be required every time:
 
-    #### 4. Setting up the cluster
+    ### Setting up the cluster
     Now that you have Kind installed, let's create a Kubernetes cluster:
 
-    1. Create a new cluster by running:
+    1. In the Terminal, copy and paste the following command and press `Enter`:
 
-      ```bash
-      kind create cluster --name odigos-demo
-      ```
-    
-    This command creates a new Kubernetes cluster named "odigos-demo".
+    ```bash
+    kind delete cluster --name odigos-demo
+    ```
+
+    This removes any existing test environment with the same name. If this is your first time, you'll see a message saying the cluster doesn't exist, which is fine.
 
-    2. Verify that the cluster is running:
+    2. Create a new cluster by running:
 
-      ```bash
-      kubectl cluster-info --context kind-odigos-demo
-      ```
-    
-      This should display information about your cluster, including the Kubernetes master and CoreDNS.
+    ```bash
+    kind create cluster --name odigos-demo
+    ```
+
+    This command creates a new Kubernetes cluster named "odigos-demo".<br />
```

Comment:
do we need the `<br /> here?

Reply:
Just for UI, it's not needed if you don't want it :)